api/client/samples/similar_regions/similar_region.py

```diff
@@ -55,7 +55,7 @@ def _format_results(self, sim_regions, region_level_id, dists):
                 grandparent = next(self.client.lookup_belongs("regions", parent["id"]), {"name": "", "id": ""})
             else:
                 grandparent = {"name": "", "id": ""}
-            self._logger.info("{}, {}, {}".format(sim_region_name, parent["name"], grandparent["name"]))
+            self._logger.info("{}, {}, {}".format(sim_region_name.encode('utf8'), parent["name"].encode('utf8'), grandparent["name"].encode('utf8')))
```

Comment:
rather than force an encoding here,  put u in front of the string constant to use the environment's encoding  `u"{}, {}, .."`

Reply:
done